compiler/src/dotty/tools/dotc/decompiler/Main.scala

```diff
@@ -14,13 +14,14 @@ object Main extends dotc.Driver {
     assert(ctx.settings.fromTasty.value)
     val outputDir = ctx.settings.outputDir.value
     if (outputDir != ".")
-      Files.deleteIfExists(Paths.get(outputDir + ".decompiled"))
+      Files.deleteIfExists(Paths.get(outputDir + "/decompiled.scala"))
     new TASTYDecompiler
   }
 
   override def setup(args0: Array[String], rootCtx: Context): (List[String], Context) = {
     var args = args0.filter(a => a != "-decompile")
-    args = if (args.contains("-from-tasty")) args else "-from-tasty" +: args
+    if (!args.contains("-from-tasty")) args = "-from-tasty" +: args
+    if (args.contains("-d")) args = "-color:never" +: args
```

Comment:
why?

Reply:
If we print colors in the output file it is impossible to read (and probably compile) the file.

Reply:
Shouldn't we do this even if `args` does not contain `-d`?

Reply:
If args does not contain `-d` the output is printed on the terminal. There it is fine and nicer to have colors. Also copy-pasting from the terminal usually removes the colors.